src/pointer/enums/Main.java

```diff
@@ -0,0 +1,97 @@
+package pointer.enums;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
```

Comment:
too complicated main, I believe can be significantly simplified but it does not matter for me right now